lib/parse.js

```diff
@@ -380,11 +377,9 @@ class Parser {
             base = nodes.numberNode(token.value, start, token.end);
         } else if (matches(token, '(')) {
             this.consume('(');
-            base = this.expression();
+            const content = this.expression();
+            base = nodes.parensNode(content, start, content.loc.end);
```

Comment:
We shouldn't be adding parentheses unconditionally... only when they are necessary.  I've added some more examples to https://github.com/semantic-math/math-parser/issues/26.